### PR TITLE
feat: Benchmark with more realistic data

### PR DESCRIPTION
**Pull request by Leonardo-Ferreira**
The original benchmark for RBAC With Cache was a bit unrealistic because the same user and the same data was always requested, which is a highly cache-able scenario. By introducing a little variance on the the user and data actually enforced, we can see a MAJOR skew on large datasets but NOT on the smaller ones (this proves that the "randomization" is not the actual at-fault for the performance drop)

### CODE REVIEW

**Comment by sagilio** [comment]
It is a great point. But the old benchmark is synced from the go casbin repo, it is better to use a new file like `EnforcerWithCacheRandomBenchmark.cs `

**Comment by Leonardo-Ferreira** [comment]
@hsluoyz this error happens because I cannot read the repos secret. If you look at the activity, it has successfully analyzed the code, but failed at the step of uploading the report due to a invalid key (it was not replaced). This issue is being fixed on a separate PR

File: Casbin.Benchmark/EnforcerWithCacheBenchmark.cs [hunk 1]
```diff
@@ -121,23 +120,31 @@ private void GlobalSetupForRbacModelWithScale(int groupCount, int userCount)
...
+                //short path)
+                var groupExt = $"group{i.ToString().PadLeft(10, '0')}";
+                var dataExt = $"data{(i / 10).ToString().PadLeft(16, '0')}";
+                //It's highly unusual for a user to be able to do only a single action,
+                //and in a real world scenarios, there are other actions such as
+                //"create", "admin", "alter", "revoke", "copy", "download",
+                //"metadata-reader" and others
+                policyList.Add(new[] { groupExt, dataExt, "read" }.ToList());
+                policyList.Add(new[] { groupExt, dataExt, "write" }.ToList());
+                policyList.Add(new[] { groupExt, dataExt, "grant" }.ToList());
```
[the remark this answers is not in the document]
Comment (Leonardo-Ferreira) [reply Casbin.Benchmark/EnforcerWithCacheBenchmark.cs:134]: That is the idea @sagilio. As mentioned on the comments above the code, imho, it's unrealistic to benchmark this kind of tool as if all the user had a single permission. It's like testing how fast your car can go on a downhill track...

[the remark this answers is not in the document]
Comment (sagilio) [reply Casbin.Benchmark/EnforcerWithCacheBenchmark.cs:134]: I know, I hope this real test can be used as a new test to compare with the original (ideal test), which at least guarantees that their total count of policies is the same.

**Comment by hsluoyz** [comment]
@sagilio

**Pull request closed without merging**